tests/game/test_buildings.py

```diff
@@ -20,6 +20,7 @@
 # ###################################################
 
 from itertools import product
+import pytest
```

Comment:
Yeah, Travis doesn't complain either. Nothing to worry though, I was just curious.